hrui/hrui.php

```diff
@@ -27,6 +27,24 @@
 
 require_once __DIR__ . DIRECTORY_SEPARATOR . 'hrui.civix.php';
 
+/**
+ * Implements hook_civicrm_coreResourceList().
+ */
+function hrui_civicrm_coreResourceList(&$items, $region) {
+  if ($region == 'html-header') {
+    if (CRM_Core_Config::singleton()->debug) {
+      CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.hrui', 'js/src/civihr-popup/attrchange.js');
+      CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.hrui', 'js/src/civihr-popup/civihr-popup.js');
+      CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.hrui', 'js/src/hrui.js');
+      CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.hrui', 'js/src/contact.js');
+    } else {
+      CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.hrui', 'js/dist/hrui.min.js');
```

Comment:
Didn't we decide to not using debug mode anymore? In that case, we can just include the min files, and let map files do its job.

Reply:
This is not an angular/requireJS app, it's a bunch of simple JS files, so i kept the original logic. There is not even any map file being created, it has nothing to do with the change in approach we made to the angular apps

Reply:
Okay, Do you think we can implement the same here too by updating the gulpfile? Then we would move towards having a single approach for all JS based extensions.